evm/vm/forks/frontier/state.py

```diff
@@ -49,96 +50,97 @@
 
 
 class FrontierTransactionExecutor(BaseTransactionExecutor):
-    def run_pre_computation(self, transaction):
+    def build_evm_message(self):
```

Comment:
What's the value here of making this rely on the internal state of `self.transaction` instead of a passed in version?